media/css/base.css

```diff
@@ -188,11 +188,11 @@ textarea {
 .left_column,
 .left_column_big {
 	float: left;
-	width: 325px;
+	/*width: 325px;*/
```

Comment:
Could we use min/max-width as hints instead?

Reply:
Hmm, I'm not convinced.

Reply:
Given that kalkun UI is not responsive. min-width will tell the browser not to break words when accessing over smaller screen device like phone/tablet. Otherwise it will try to fit everything causing the UI to break.

Reply:
Kalkun has a fixed width anyway so on small devices it looks like on desktop but highly unzoomed. I tried with duckduckgo app.

Because of the width that was there, the UI already broke on desktop. Putting min-width won't change things because the min-width will act as set width in that case. (On mobile it was ok).

Still I'm not convinced.

Maybe you will have a better understanding with screnshots
Before:
![Screenshot_20220228_122138](https://user-images.githubusercontent.com/46226844/155975266-db025467-38fa-4eac-ad73-b9476448fc04.png)

After:
![Screenshot_20220228_122202](https://user-images.githubusercontent.com/46226844/155975273-26ba3ad9-9926-4e04-b8c2-944a326377d9.png)

Reply:
Ah I see. Makes sense